Dockerfile

```diff
@@ -7,30 +7,52 @@ LABEL maintainer="Pedro Sanders <psanders@fonoster.com>"
 #   --build-arg FONOSTER_VERSION=0.2.20 \
 #   --tag fonoster/fonoster:0.2.20 .
 
-ARG FONOSTER_VERSION=0.2.20
-ARG BRANCH=main
-ENV FONOSTER_VERSION=$FONOSTER_VERSION
-ENV BRANCH=$BRANCH
-
 WORKDIR /work
 
-RUN apk add --no-cache --update git curl docker docker-compose openssl bash\
-  && git clone https://github.com/fonoster/fonoster --depth=1 -b $BRANCH --single-branch \
-  && curl -qL -o /usr/bin/netdiscover https://github.com/CyCoreSystems/netdiscover/releases/download/v1.2.5/netdiscover.linux.amd64 \
-  && chmod +x /usr/bin/netdiscover \
-  && mkdir -p operator config \
-  && touch config/config config/user_credentials \
-  && cp -a fonoster/operator/compose/* operator \
-  && mv operator/env_example operator/.env \
-  && cp fonoster/etc/rbac.json config \
+# Copy project files to /work
+COPY . /work/fonoster
+
+# Install dependencies
+RUN apk add --no-cache --update git curl docker docker-compose openssl bash
```

Comment:
Please refactor to join as many RUN steps as possible. Joining the steps will help reduce the size of the resulting image.

Reply:
Fixed, I get the same image size as always but using multiple stages. @psanders